src/index.html

```diff
@@ -17,6 +21,94 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="header-container">
+        <div class="logo-container">
+          <a href="#">
+            <img
+              src="./images/logo.png"
+              alt="MOYO Logo"
+              class="logo"
+            />
+          </a>
+        </div>
+        <nav class="nav__container">
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. There should be an empty line before and after the 'nav' element.